README.md

```diff
@@ -46,24 +46,25 @@ This project was made with
 ## Prerequisities
 1. Python 3.9.7+
 2. Required LOINC Files
-For now, these files are [downloadable from GoogleDrive](https://drive.google.com/drive/u/0/folders/1SjDFYs1ocbpovGlAZDKuRVcTDoNztHOc). They need to be put in these locations:
+They need to be put in these locations:
 - `data/part_files/ComponentTree.tsv`
 - `data/code_files/LOINC.csv`
 - `data/code_files/LoincPartlink_Primary.csv`
 
 
 ## Setup
 1. Clone this repository.
-2. install poetry
-3. run poetry install
+2. install poetry [pip install poetry]
+3. run `poetry install`
+4. run `poetry shell`
 
 ## Usage
 Commands 1.1 - 1.5 are meant to be run sequentially.
 
-Alternatively, you can run all of them at once using default values by running `python src/comp_loinc/cli/main.py all`.
+Alternatively, you can run all of them at once using default values by running `python src/comp_loinc/main.py all`.
```

Comment:
The rest of these changes, I am not sure if they are still applicable because this PR is old. `main` is older, but IDK if Shahim has something that needs to be merged which might invalidate these changes. I'll confer with him.